feat: i18n for jupyter

Jose-Matsuda opened the pull request:
Resolves #48 
-Install the French language locale in the environment
-Upon notebook starting, the locale from the argv file is taken and used as the "Language" ENV variable.
-For changes to be applied, the onload of the landing page must be hit to trigger a restart of jupyter.
-The onload handles the changing of the locale in argv (vscode changes)
-Currently the browser must be in fr_FR for **everything** to get translated
 In fr_CA, a few widgets in jupyter notebook are not translated like "Save" and "Run"
-This is weird to me as all translations seemingly come from the [nbui](https://github.com/jupyter/notebook/blob/master/notebook/i18n/fr_FR/LC_MESSAGES/nbui.po)  from fr_FR/LC_MESSAGES in jupyter notebook. 
The widget values exist in that file, but in fr_CA they do not get applied.
Comparing this to say "Clusters tab..." which **does get translated in fr_CA** this makes less sense to me

This contains changes from my branch for VSCODE, when that is merged in to master I can rebase this off of that. 
The changes are to `languagecheck.js` (new file from vscode changes) and `start-notebook.sh`, as well as the Dockerfile to install the French locale

[Video demonstrating France French and Canadian French](https://youtu.be/QPsvuDbu3mY)

Jose-Matsuda commented:
For this review, only [start-notebook.sh](https://github.com/StatCan/kubeflow-containers-desktop/pull/52/files#diff-8bf6b86b192814b359d0c2c2dabc6770792c66b6b6a3ab6d61b50f852bfaf9e2R7), the installation of the French locale in the [dockerfile](https://github.com/StatCan/kubeflow-containers-desktop/pull/52/files#diff-b093d34eec92adf6e0cc6e445edb7147ea45d25fb0c98f2f84a60f5aae6f34f7R742) and the jupyter notebook reset call in [languagecheck.js](https://github.com/StatCan/kubeflow-containers-desktop/pull/52/files#diff-4e51a8478792f4d3fa9cc96fc2d69741b99486f1e4c07b46f417420ac37892a9R34) from the VScode changes need to be looked at. Everything else is the same (pre Brendan review changes) as the PR for vscode

Jose-Matsuda commented:
i closed by accident wait up

Jose-Matsuda commented:
wait for the merging of the other branch (Microsoft branch fix)

wg102 commented on base/Dockerfile:
```diff
@@ -754,6 +754,16 @@ ARG START_RSYSLOGD="/usr/sbin/rsyslogd -n"
     echo "${NB_USER} ALL=(ALL) NOPASSWD: ${START_CRON}, ${START_NETDATA}, ${START_RSYSLOGD}" >> /etc/sudoers.d/${NB_USER}
 
+# Install the French Locale
+RUN \
```
I suggest to add a small comment on why we're using fr_FR instead of fr_CA

Jose-Matsuda replied:
My mistake, Only browser needs to be in fr_FR. Installed locale can be fr_CA

wg102 commented on base/resources/scripts/languagecheck.js:
```diff
@@ -30,34 +30,74 @@ http
+      res.end(responseText);
+    }
+
+    //Used to kill the jupyter process, supervisord will restart it
```
Typo

wg102 reviewed (changes requested):
Two small changes in comments but otherwise it looks good

Commit:
feat: i18n for base Jupyter
-install the French France locale
--Browser must be fr_FR for some widgets' hover-over to be translated
--Jupyter only has translations for fr_FR
--See video in PR comment
-Use value from vscode's argv file to determine what lang to start in
-jupyter process restarts on landing page iff browser language changes
